### DISCUSSION TITLE
bg-white dark:bg-transparent not working

### DISCUSSION THREAD

**Discussion by test-n-prod** [tailwindlabs/tailwindcss#11467] [Help]
similar to https://github.com/tailwindlabs/tailwindcss-jit/issues/27, the background doesn't change to transparent while in dark mode. this works fine if i do bg-white dark:bg-black, or any other color

similarly, bg-white dark:bg-opacity-0 results in no difference between dark and light mode

**Comment by wongjn** [comment]
It seems to work in [Tailwind Play](https://play.tailwindcss.com/Vl1vuLhy7G), which suggests it could be some misconfiguration or misuse. Consider providing a reproduction so that we can troubleshoot the unexpected behavior.
